core/dbt/config/runtime.py

```diff
@@ -297,17 +297,18 @@ def from_parts(
 
     @classmethod
     def from_args(cls: Type[RuntimeConfig], args: Any) -> 'RuntimeConfig':
-        """Given arguments, read in dbt_project.yml from the current directory,
-        read in packages.yml if it exists, and use them to find the profile to
-        load.
+        """Given arguments, read in dbt_project.yml from the arg --project-dir
+        or current directory if not provided, read in packages.yml if it exists,
+        and use them to find the profile to load.
```

Comment:
```suggestion
        or current directory if not provided, read in packages.yml if it
        exists, and use them to find the profile to load.
```
This will appease flake8!

Reply:
Thanks for the catch! accepted the suggestion :+1: